### PR TITLE
Don't `gzip` in PHP, and add constant for CSS minification

### PR DESCRIPTION
**Pull request by aidvu**
- `gzip` in PHP slows stuff down a bit. Simply don't do this. Any web server can handle this better.
- also remove the output buffering, no need for that anymore
- CSS Minification can sometimes slow things down significantly. Add constant to enable/disable.

### CODE REVIEW

File: page-optimize.php [hunk 1]
```diff
@@ -17,6 +17,10 @@
 	define( 'PAGE_OPTIMIZE_ABSPATH', ABSPATH );
 }
 
+if ( ! defined( 'PAGE_OPTIMIZE_CSS_MINIFY' ) ) {
```
Comment (aidvu) [inline page-optimize.php:20]: Disable by default. Not much benefit, slows things a lot.

File: service.php [hunk 2]
```diff
@@ -86,7 +86,7 @@ function page_optimize_service_request() {
 
 function page_optimize_build_output() {
 	global $page_optimize_types;
-	ob_start( 'ob_gzhandler' );
+	ob_start();
```
Comment (brandonpayton) [inline service.php]: I think we can just skip output buffering and return the content at the end like:
```php
'content' => $pre_output . $output,
```

Comment (aidvu) [reply service.php]: Right! No need for buffering now that we don't `gzip`.

**Review by brandonpayton — APPROVED** [review]
Looks good! Thanks!

**Comment by brandonpayton** [comment]
In a separate PR, it might be worth checking if replacing string concat throughout `page_optimize_build_output()` with output buffering and echo would be faster. According to this very old StackOverflow answer, it was faster in 2010:
https://stackoverflow.com/a/6934825

If you don't get to it, I can try it out tomorrow. 🙇

File: service.php [hunk 3]
```diff
@@ -5,9 +5,6 @@
 	'js' => 'application/javascript'
 );
 
-// TODO: Provide a settings button to clear the cache
```
Comment (aidvu) [inline service.php:8]: It auto clears, and with new files is always fresh. Don't think it's needed.

File: service.php [hunk 4]
```diff
@@ -5,9 +5,6 @@
...
-// TODO: Should we provide a default at all? Is this a reasonable default?
```
Comment (aidvu) [inline service.php:9]: This was prolly for the cache, not needed anymore.

**Pull request merged**